src/main/java/org/sagebionetworks/web/client/widget/entity/MarkdownEditorWidgetViewImpl.java

```diff
@@ -385,4 +388,9 @@ public void setVideoCommandsVisible(boolean visible) {
 		videoLink.setVisible(visible);
 		videoButton.setVisible(visible);
 	}
+
+	@Override
+	public void setExternalImageButtonVisible(boolean visible) {
+		externalImageButton.setVisible(visible);

```

Comment:
I realize it's not currently necessary (since this is set in the ui.xml), but could you add externalImageButton.setVisible(false) in the view.clear() method for me?